app/src/gui/components/alert.tsx

```diff
@@ -14,29 +14,18 @@
  * limitations under the License.
  *
  * Filename: alert.tsx
- * Description:
- *   TODO
+ * Description: The SystemAlert component displays alert messages in the form of snackbars at the bottom-center of the screen.
+ *   Alerts are shown one at a time with configurable durations and severity levels.
  */
 
-import React, {useContext} from 'react';
+import React, { useContext } from 'react';
```

Comment:
Looks like an issue with your prettier config.

Reply:
Sure, updating the prettier fixes in the PR